website/versioned_docs/version-22.2.2/CLI.md

```diff
@@ -0,0 +1,328 @@
+---
+id: version-22.2.2-cli
```

Comment:
Can we use just major and minor versions for docs? I'd rather have docs for "Jest 22.2". Patches are for fixes

Reply:
Definitely, pushing a fix 👍

Reply:
22.2.X seems reasonable

Reply:
I can see on React Native's website (https://facebook.github.io/react-native/versions.html) that they use just major and minor versions (`0.53`), skipping the patch version (like @thymikee suggested).

Looking at it again I think it looks better without the `X`, what do you guys think?